### PR TITLE
[UXIT-1706] Enhance Clarity and Consistency in ‘What is Filecoin’ Section

### PR DESCRIPTION
**Pull request by mirhamasala**
This PR improves readability and consistency in multiple sections of the Filecoin documentation, including updates to language flow, terminology alignment, and clearer explanations in key areas such as the storage model, retrieval market, blockchain, crypto-economics, networks, and programming on Filecoin.

### CODE REVIEW

**Review by smagdali — CHANGES_REQUESTED** [review]
This is great. I'm definitely not a fan of American-style 'Capitalising All Headings No Matter What' though, and that's not what we do elsewhere on the site? 

Huh, the readme.md on github used to have a style guide in it, I guess Johnny deleted it when we moved to Gitbook.

File: basics/what-is-filecoin/blockchain.md [hunk 1]
```diff
@@ -1,119 +1,111 @@
...
-* Storage Miner Actor - storage mining operation and validate storage proofs.
-* Storage Power Actor - keeping track of the storage power allocated at each storage provider
-* Storage Market Actor - managing storage deals.
-* Multisig Actor - responsible for operations involving the Filecoin multi-signature wallet.
-* Payment Channel Actor - set up and settle payment channel funds.
-* Datacap Actor - responsible for datacap token management.
-* Verified Registry Actor - responsible for managing verified clients.
-* Ethereum address Manager (EAM) Actor- responsible for assigning all Ethereum compatible addresses on Filecoin Network, including EVM smart contract addresses and Ethereum account addresses.
-* EVM Account Actor - a non-singleton built-in actor representing an external Ethereum identity backed by a secp256k1 key.
+- **System Actor**: General system actor.
```
Comment (smagdali) [inline basics/what-is-filecoin/blockchain.md:22]: These should all be normal case. "Init actor" - they are not proper nouns (and yes, this is the only one that was correct before)

File: networks/calibration/explorers.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ description: >-
 
 ## FILFOX
 
-[https://calibration.filfox.info/en](https://calibration.filfox.info/en)
+[https://calibration.filfox.info/en](http://47.109.105.51/en)
```
Comment (smagdali) [inline networks/calibration/explorers.md:15]: this situation is temporary - I'm trying to figure out what's going on with them but we should probably list both for now.

Comment (mirhamasala) [reply networks/calibration/explorers.md:15]: It doesn't pass the link checker test if the link is broken and thus the PR can't be merged.. We can revert is once they fix it?

**Comment by mirhamasala** [comment]
@smagdali - I quite like the visual hierarchy that capitalising heading provides. And was planning for us to implement it throughout the docs. Following your comment, I reverted it.

> This is great. I'm definitely not a fan of American-style 'Capitalising All Headings No Matter What' though, and that's not what we do elsewhere on the site?

> Huh, the readme.md on github used to have a style guide in it, I guess Johnny deleted it when we moved to Gitbook.

Yes, a style guide would be helpful.

File: basics/what-is-filecoin/blockchain.md (hunk 1 above)
Comment (mirhamasala) [reply basics/what-is-filecoin/blockchain.md]: Okay - done.

**Pull request merged**